core/src/main/java/org/commonjava/maven/ext/core/impl/DependencyManipulator.java

```diff
@@ -430,6 +432,12 @@ private void apply( final Project project, final Model model, final Map<Artifact
                     dependencyManagement.getDependencies().addAll( 0, extraDeps );
                 }
             }
+            else if ( commonState.isOverrideTransitive() && dependencyState.getRemoteBOMDepMgmt() == null )
+            {
+                logger.warn( "Ignoring {}={} since it was used without the {} option",
+                        CommonState.TRANSITIVE_OVERRIDE_PROPERTY, commonState.isOverrideTransitive(),
+                        DependencyState.DEPENDENCY_MANAGEMENT_POM_PROPERTY );
```

Comment:
I still think that parametrising a string to combine two strings in the end is rather pointless and seems unnecessary - and potentially it less performant than a single string. I don't think this needs any parameters.

Reply:
[ see conversation above ]